src/ca/ubc/cs304/model/Branch.java

```diff
@@ -0,0 +1,19 @@
+package ca.ubc.cs304.model;
+
+public class Branch {
+    private String location;
+    private String city;
+
+    public Branch(String location, String city) {
+        this.location = location;
+        this.city = city;
+    }
+
+    public String getLocation() {
```

Comment:
Do we need to make setters for these? How are we transferring data back to the database? via a model I'd imagine